### ISSUE TITLE
How can i obtain all words from given dictionary element

### ISSUE THREAD

**Issue by MonsterMMORPG** [hunspell/hunspell#473]
I would like to parse open office supporting hunspell formatted aff and dic files.

English aff and dic files can be downloaded from here for example : http://extensions.openoffice.org/en/project/english-dictionaries-apache-openoffice

I want to scan each line of the given .dic file and generate every possible word of the each line with the provided .aff file

How can i do that?

I have installed NHunspell framework but it does not have that feature : https://www.nuget.org/packages/NHunspell/

For example for the english language lets consider

make/UAGS
make can be make, made, makes, making etc

Now i need parser to give me all these combinations. How can i obtain them? Ty very much

So basically i want to scan each line of the dictionary and generate all possible words from the word of that line and i dont know how can i do that

I can also write my own parsers, but it seems to me rules are pretty complex and there are no detailed and easy documentation about this

Here what i want basically. The image explains very clearly

Giving analyze/ADSG, en.dic and en.aff file and obtaining all the following words

analyze, analyzes, analyzing, analyzed, reanalyze, reanalyzes, reanalyzing, reanalyzed

![what i want](https://cloud.githubusercontent.com/assets/4149474/23531335/ebc1af9a-ffb6-11e6-8aba-f44891e4a32e.png)

**Comment by dimztimz** [comment]
Try either with:
1. unmunch (a C++ program)
2. wordforms (Bash script)

You can find both under https://github.com/hunspell/hunspell/tree/master/src/tools

**Comment by MonsterMMORPG** [comment]
@dimztimz  how do i use bash script? Can you show me example? I also need to write it to the file
